sysutils/pfSense-pkg-node_exporter/files/usr/local/share/pfSense-pkg-node_exporter/interface-collector.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
Just a note that you'll need to try this python script on 2.5.0 snapshots (python 3.6) and 2.4.x (python 2.7) to ensure it works in both environments.

Reply:
Validated that the script works under both python versions. Updated with two suggestions from `2to3` (although they don't seem to be strictly needed, it worked as-was).
I don't have a spare pfsense environment to test pf2.5 on, however. Are there any changes to the config.xml? The script does some parsing of the `interfaces` tree.

Reply:
Nothing changed around interfaces, so it should be OK